lib/active_model/serializer/associations.rb

```diff
@@ -29,7 +28,8 @@ class << base
 
       module ClassMethods
         def inherited(base)
-          base._reflections = self._reflections.try(:dup) || []
```

Comment:
Module `Associations` defines (on the class that includes it) a class attribute `_reflections` that is initialized to `[]` and is inherited.